extension/src/shared/view/selection/DescriptionFeatureContent.tsx

```diff
@@ -90,7 +103,6 @@ const Wrapper = styled("div")`
   width: 100%;
   min-width: 100%;
   box-sizing: border-box;
-  padding: 5px;
```

Comment:
This might affect for this data `小倉駅周辺の群衆度（Wi-Fi電波）`. We should keep margin for this part.
![Screenshot 2024-03-01 at 15 18 07](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/assets/34934510/f5f5edee-c477-4e35-b6d7-2f401f316a92)